app/views/articles/search/_sidebar.html.erb

```diff
@@ -12,6 +12,9 @@
         <a class="query-filter-button" data-filter="class_name:Comment">COMMENTS</a>
         <hr>
         <a class="query-filter-button my-posts-query-button" data-filter="MY_POSTS">ONLY MY POSTS</a>
+        <hr>
+        <h4>sort by</h4>
+        <%= select_tag "sort", @sort_options, include_blank: true, class: "query-filter-select" %>
```

Comment:
Why do we include blank option here?

Reply:
To be perfectly honest, when I took it out, the select dropdown stopped working entirely - like the dropdown wouldn't appear when you clicked it - and I could not figure out why. I'll try and poke at it a bit more tomorrow.

Reply:
Once I moved the sort options to the view this stopped being an issue.